src/main/java/org/junitpioneer/jupiter/CartesianProductTest.java

```diff
@@ -72,10 +74,19 @@
 		private final List<List<?>> sets = new ArrayList<>(); //NOSONAR
 
 		public Sets add(Object... entries) {
-			sets.add(new ArrayList<>(Arrays.stream(entries).distinct().collect(toList())));
+			sets.add(toDistinctList(Arrays.stream(entries)));
 			return this;
 		}
 
+		public Sets addAll(Collection<?> entries) {
```

Comment:
Didn't you say we should be able to add a `Stream` too...?

Reply:
Uh, no? 🤷‍♂️ You mean `addAll(Stream<?>)` in addition?

Reply:
Yes, the option is already there, just make it `public` and rename it to `add`/`addAll`. 😄

Reply:
Should we add the possibility to documentation, with a small code example?